cypress/e2e/products/productsList/filteringProducts.js

```diff
@@ -24,7 +24,7 @@ import {
 } from "../../../support/pages/catalog/products/productsListPage";
 
 describe("As an admin I should be able to filter products", () => {
-  const startsWith = "CyFilterProducts-";
+  const startsWith = "ACyFilterProducts-";
```

Comment:
**question**: why filters now start with `ACy`?

Reply:
The point is we have a snap of DB (refreshed daily) which makes our tests fail - filters have various names - and when displaying new filters we use alphabetical order - so this is a very easy fix to make a newly created filter be displayed as first among other records from DB